### PR TITLE
accumulate: Correct the readme

### LINKED ISSUE #1070: accumulate: Readme doesn't match the tests.
The Readme for accumulate uses an example of a numeric list, but the test cases all use strings. The Readme should be rewritten. 

----
Given the collection of strings:

    "the", "cat", "sat", "on", "the", "mat"

And the operation:

`Capitalise` a string (`"cat" => "CAT"`)

Your code should be able to produce the collection of words:

    "THE", "CAT", "SAT", "ON", "THE", "MAT"
----

### PR DESCRIPTION
**Pull request by ooransoy**
Fixes #1070.

### CODE REVIEW

File: exercises/accumulate/README.md [hunk 1]
```diff
@@ -29,10 +29,10 @@ Solve this one yourself using other basic tools instead.
 
 To run the tests run the command `go test` from within the exercise directory.
 
-If the test suite contains benchmarks, you can run these with the `--bench` and `--benchmem`
-flags:
+If the test suite contains benchmarks, you can run these with the `-bench`
+flag:
```
Comment (leenipper) [inline exercises/accumulate/README.md]: Did you notice that the differences shown dropped some standard lines for the Go READMEs here? The standard lines come from [here](https://github.com/exercism/go/blob/master/config/exercise_readme.go.tmpl#L11-L14).
And, the README.md files are generated using a tool called configlet, that gathers pieces from standard places, track specific,  and also from the .meta directory (if applicable) under each exercise.  For a few exercises, the base content of the README.md comes from a .meta/description.yml file under the exercise directory. For example, see exercises/yacht; and in those cases, the .meta directory must also include a copy of the metadata.yml from the problem-specifications/exercises/.../metadata.yml (without change).
First, clone the problem-specifications repo, at the same level, directory-wise, as your local exercism/go repo.
Use exercises/accumulate/description.md from that repo to make your adjustments (numbers vs. strings) and put the result in exercises/accumulate/.meta/description.md; copy the metadata.yml for yacht from problem-specifications/exercises/accumulate/metadata.yml into the new accumulate/.meta directory
Then get a up-to-date copy of the configlet tool:
```
./bin/fetch-configlet
```
Then, run configlet:
```
./bin/configlet help generate
./bin/configlet generate --only accumulate
```
The README.md generated for accumulate should show only the specific changes you made for the strings vs. numbers and that text would be in the exercises/accumulate/.meta/description.md.

File: exercises/accumulate/README.md [hunk 2]
```diff
@@ -29,10 +29,10 @@ Solve this one yourself using other basic tools instead.
...
 
-    go test -v --bench . --benchmem
+    go test -bench .
```
Comment (leenipper) [inline exercises/accumulate/README.md]: And here too.  Lost the standard track piece.

**Comment by bitfield** [comment]
Happy to pick this up, @sebito91, if the OP isn't available.

**Comment by kytrinyx** [comment]
@bitfield Since we've not heard back for almost 5 months, I think the right move here is to carry it!

**Comment by jrake-revelant** [comment]
+1 I ran into this problem

**Pull request merged**